### PR TITLE
Feature/add linters

### PR DESCRIPTION
**Pull request by DZariusz**

### CODE REVIEW

**Comment by enddynayn** [comment]
👍

**Comment by enddynayn** [comment]
nice! thank you.

File: contracts/Plasma.sol [hunk 1]
```diff
@@ -153,55 +155,57 @@ contract Plasma is Ownable {
 
   // challenge
   function challengeWithdrawDeposit(uint _blockIndex, bytes _transactionBytes, bytes _proof, bytes signature) public returns (bool success) {
-    Block memory block = blocks[_blockIndex];
+    Block memory newBlock = blocks[_blockIndex];
```
Comment (avoloshko) [inline contracts/Plasma.sol]: it's not new. it's a block where the deposit has been spent.

File: contracts/Plasma.sol [hunk 2]
```diff
@@ -153,55 +155,57 @@ contract Plasma is Ownable {
...
   // temp methods
-  function verifyProof(bytes _proof, bytes32 _root, bytes32 _leaf, uint256 _index) public constant returns (bool success) {
+  function verifyProof(bytes _proof, bytes32 _root, bytes32 _leaf, uint256 _index) public pure returns (bool success) {
     return MerkleProof.verifyProof(_proof, _root, _leaf, _index);
   }
 
-  function proveTX(uint _blockIndex, bytes _transactionBytes, bytes _proof) public constant returns (bool success) {
-    Block memory block = blocks[_blockIndex];
+  function proveTX(uint _blockIndex, bytes _transactionBytes, bytes _proof) public view returns (bool success) {
+    Block memory newBlock = blocks[_blockIndex];
```
Comment (avoloshko) [inline contracts/Plasma.sol]: why 'new'?

Comment (DZariusz) [reply contracts/Plasma.sol]: because `block` is global variable in solidity and compiler give us a warning for it.

**Comment by therevoltingx** [comment]
@DZariusz 
Let's pair up and fix the issues with codeship and get this merged!

**Pull request merged**